### DISCUSSION TITLE
Refresh live preview when changing a Matrix block's status in element index view

### DISCUSSION THREAD

**Discussion by ryansupercool** [craftcms/cms#16057] [Ideas] [answered]
In Craft 5, given a Matrix field is using the "As an element index" view mode, would it be possible to refresh live preview when the blocks status is changed from enabled/disabled?

**Comment by brandonkelly** [comment, marked as answer]
Yep! Just made that the case for the next release. (3212790cae371bc5693eb33ba902f8024a031bd1)

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with that change 🎉
